src/selectors/visibleColumnBreakpoints.js

```diff
@@ -101,10 +102,20 @@ export function getColumnBreakpoints(pausePoints, breakpoints, viewport) {
     ({ location: { line } }) => lineCount[line] > 1
   );
 
-  return columnBreakpoints.map(({ location }) => ({
-    location,
-    enabled: isEnabled(location, breakpointMap)
-  }));
+  return columnBreakpoints.map(({ location }) => {
+    const foundBreakpoint = breakpoints.find(breakpoint => {
+      return (
+        breakpoint.location.line === location.line &&
+        breakpoint.location.column === location.column
+      );
+    });
+
+    return {
+      location,
+      enabled: isEnabled(location, breakpointMap),
+      condition: foundBreakpoint ? foundBreakpoint.condition : ""
```

Comment:
Should we default this to `null`? I see the type is `?string`, but it's not clear if there is a difference between empty string and null/undefined here.